nsv13/code/modules/food_and_drinks/food/custom_food.dm

```diff
@@ -0,0 +1,11 @@
+/obj/item/reagent_containers/food/snacks/hardtack //NSV13
+	name = "hardtack"
+	desc = "Traditional pirate snack. Legally considered a dangerous weapon."
+	icon_state = "hardtack"
+	bonus_reagents = list(/datum/reagent/consumable/nutriment = 1)
+	list_reagents = list(/datum/reagent/consumable/nutriment = 2)
+	tastes = list("broken teeth" = 7,"depression" = 4,"flour and water" = 5)
+	foodtype = GRAIN
+	force = 9
```

Comment:
I suspect you wanted to use `throwforce` for this, or possibly just also add it.